android/source/VideoLocker/res/layout/activity_find_courses.xml

```diff
@@ -0,0 +1,36 @@
+<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/drawer_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/grey_act_background"
+    android:splitMotionEvents="false" >
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <View
+            android:id="@+id/blue_bar"
```

Comment:
I'm not 100% sure, but I believe the blue bar is part of the web content and so we don't need to handle it here. Let's remove it for now and re-add it if necessary.